tests/integration/dbclient/appl/pom.xml

```diff
@@ -0,0 +1,417 @@
+                                <include>io.helidon.tests.integration.dbclient.appl.it.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.simple.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.transaction.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.statement.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.metrics.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.result.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.mapping.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.interceptor.*IT</include>
+                                <include>io.helidon.tests.integration.dbclient.appl.it.health.*IT</include>
+                            </includes>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+
+    </build>
+
+    <profiles>
+
+        <profile>
+            <id>docker</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>start</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <goal>start</goal>
+                                </goals>
+                            </execution>
+                            <execution>
+                                <id>stop</id>
+                                <phase>post-integration-test</phase>
+                                <goals>
+                                    <goal>stop</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>native-image</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <native.image>true</native.image>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
```

Comment:
Can you remove these profiles and instead document manual docker steps in the README ?

Reply:
Those profiles contain database setup and they are used for both local and global tests runs including docker image initialization for local run.
How would you resolve DB specific dependencies in POM without profiles?

Reply:
Well, this is something I don't want to do. Even with all this docker and DB profiles testing of all supported DBs and JVM/native image modes takes almost an hour. It's something I have to do with every PR update when working on integration tests.
Manual setup will make this step much worse for anyone who will have to work on this in the future.

Reply:
IMO this is only for your convenience, doing this with Maven is horrible. Just add a shell that does exactly what you need. We should remove the other stuff you did to make sure it's not a pattern.

Reply:
Removed.